.percy.js

```diff
@@ -16,6 +16,10 @@ module.exports = {
         include: /(.*-smoke-test.*)\/components\/rss-feeds\/index\.html/,
         waitForSelector: 'table[data-table-name="rss-feed"]',
       },
+      {
+        include: /(.*-smoke-test.*)\/components\/mermaid-diagram\/index\.html/,
+        waitForTimeout: 2000,
```

Comment:
@timonrey  I don't think you need to guess with a timeout here.  You can use the same approach as with the RSS feed and wait for a selector.  
For example  `#body-content svg.statediagram`   try here: https://commercetools-docs-kit.vercel.app/docs-smoke-test/components/mermaid-diagram

Reply:
Ah yes, thanks. I didn't see the that there is a class defined there. 👍🏼